pkg/utils/utils.go

```diff
@@ -0,0 +1,123 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I really hate bloating utils.go modules. Could you please split it into one module for the VF handling and one for generic cache? Or at least rename it to something that would suggest this utils module is VF specific.

Reply:
Yes, makes sense. moved into sriov package having sriov and cache go modules.